base_tier_validation_formula/models/tier_definition.py

```diff
@@ -29,7 +29,11 @@ class TierDefinition(models.Model):
     def onchange_review_type(self):
         super(TierDefinition, self).onchange_review_type()
         self.reviewer_expression = (
-            "# Available locals:\n"
-            "#  - rec: current record\n"
-            "#  - Expects a recordset of res.users"
+            (
+                "# Available locals:\n"
+                "#  - rec: current record\n"
+                "#  - Expects a recordset of res.users"
+            )
```

Comment:
why the double parenthesis? you could just do `"some text" if condition = True else False`

Reply:
done